docs/hsi.xml

```diff
@@ -259,6 +267,38 @@
       </para>
     </refsect2>
 
+    <refsect3 id="org.fwupd.hsi.Uefi.SecureBoot">
+      <title>UEFI SecureBoot</title>
+      <para>
+        UEFI Secure boot is a verification mechanism for ensuring that code
+        launched by firmware is trusted.
+      </para>
+      <para>
+        Secure use of UEFI Secure Boot requires that each binary loaded at
```

Comment:
I don't think you need to use "Secure use of UEFI Secure Boot".  Just "UEFI secure boot"

Reply:
Fixed.